### DISCUSSION TITLE
Support negative margin of scrollbar

### DISCUSSION THREAD

**Discussion by Amazingkenneth** [hecrj/iced#1664] [Ideas]
In #1632, the scrollbar blocks the text in `iced::widget::scrollable::Scrollable`. This problem may be solved by setting the margin to the inverse of maximum character width. In this way, the scrollbar with the negative margin makes room for the text in the window.

**Comment by 13r0ck** [comment]
https://github.com/iced-rs/iced/issues/1632#issuecomment-1370259831
